extendr-api/Cargo.toml

```diff
@@ -15,12 +15,12 @@ license = "MIT"
 repository = "https://github.com/extendr/extendr"
 
 [dependencies]
-libR-sys = "0.3.0"
-extendr-macros = { path = "../extendr-macros", version = "0.3.1" }
-extendr-engine = { path = "../extendr-engine", version = "0.3.1" }
-ndarray = { version = "0.15.3", optional = true }
+libR-sys = { workspace = true }
```

Comment:
Can you explain what you're doing here with the internal dependencies?

Reply:
I need to use `libR-sys` in the `extendr-macro`. There is no need to create version confusion, so instead I make it a workspace dependency and use that all around when needed. This is how that is done.